pkg/asset/manifests/capiutils/manifest.go

```diff
@@ -13,8 +13,9 @@ const (
 
 // GenerateClusterAssetsOutput is the output of GenerateClusterAssets.
 type GenerateClusterAssetsOutput struct {
-	Manifests         []*asset.RuntimeFile
-	InfrastructureRef *corev1.ObjectReference
+	Manifests          []*asset.RuntimeFile
+	InfrastructureRef  *corev1.ObjectReference
+	InfrastructureRefs []*corev1.ObjectReference
```

Comment:
Just noticed this now, sorry. Maybe we could make this always an array instead of having 2 fields with very similar names?

Reply:
So I can do that, but I'll have to touch all the other platforms.  I'm fine with that if you are ok with that.  All other platforms return single right now.

Reply:
@patrickdillon can you ack you're ok with this change?

Reply:
Patrick gave me verbal (slack) approval to make this change.